tuned-main.conf

```diff
@@ -22,9 +22,10 @@ update_interval = 10
 # one hardcoded profile (by default "balanced").
 recommend_command = 1
 
-# Whether to reapply sysctl from the e.g /etc/sysctl.conf, /etc/sysctl.d, ...
-# If enabled these sysctls will be re-appliead after Tuned sysctls are
-# applied, i.e. Tuned sysctls will not override system sysctls.
+# Whether to reapply sysctl from /etc/sysctl.conf, /etc/sysctl.d and
```

Comment:
I'd change the order of directories/files to reflect the order in which the application of the sysctls is being performed.  I.e.: something like: 

Whether to reapply user-provided sysctl settings from `/run/sysctl.d/*.conf`, `/etc/sysctl.d/*.conf` and `/etc/sysctl.conf`.

Reply:
Thanks! Done.